models/recipe.js

```diff
@@ -31,6 +32,22 @@ module.exports = (sequelize, DataTypes) => {
   Recipe.associate = (models) => {
     Recipe.belongsTo(models.User, {
       onDelete: 'CASCADE',
+      allowNull: false
+    });
+    Recipe.belongsToMany(models.Plan, {
+      through: 'Plan_recipe',
+      onDelete: 'CASCADE',
+      defaultValue: null
+    });
+    Recipe.belongsToMany(models.Measure, {
```

Comment:
Here you're saying that the `Recipe` has many `Measures`, I think it's the `recipe_ingredient` the one that has a measure.